config/packages.csv

```diff
@@ -0,0 +1,28 @@
+"ICER","spablotemporal/ICE-R"
+"dampack","DARTH-git/dampack"
+"simpleMCM","Phillia/SimpleMCM"
+"dampack","feralaes/dampack"
+"vandet","n8thangreen/VANTDET"
+"cedecisiontree","Health-Economics-in-R/Cedecisiontree"
+"ceplot","Health-Economics-in-R/ceplot"
+"qaly","Health-Economics-in-R/QALY"
+"heromod2","jrdnmdhl/heRomod2"
+"dcurve","ddsjoberg/dcurves"
+"descem","Roche/descem"
+"dcea","miqdadasaria/DCEA"
+"aldvmm","pletschm/aldvmm"
+"hedtree","petedodd/Hedtree"
+"tuu","ready4-dev/TTU"
+"dalycalculator","TuftsCEVR/DALYcalculator"
+"ceact","ielbadisy/CEACT"
+"rrapidmarkov","EdCFWilson/rrapidmarkov"
+"hervis","dark-peak-analytics/heRvis"
+"survHE","giabaio/survHE"
+"survHEhmc","giabaio/survHEhmc"
+"rpsftmPlus","Roche/rpsftmPlus"
+"survdigitzerR","Pechli-Lab/SurvdigitizeR"
+"maic","Roche/MAIC"
+"flexsurvPlus","Roche/flexsurvPlus"
+"gemtc","Roche/gemtcPlus"
+"youthu","ready4-dev/youthu"
+"bimp","ready4-dev/bimp"
```

Comment:
nb this package list got truncated after some package names contained invalid characeters